### PR TITLE
Forward/rel/6.3

### PR DESCRIPTION
**Pull request by bennettbuchanan**

### CODE REVIEW

**Comment by bennettbuchanan** [comment]
@ironman-machine go

**Comment by electrachong** [comment]
@ironman-machine double-check after changing order of properties in serialization

**Comment by bennettbuchanan** [comment]
end-to-end passed. Unit tests was altered to accommodate changes with commit https://github.com/scality/S3/pull/498/commits/394f630eca033760d1af9b9bb1d4c1c0e286944e

File: lib/metadata/ModelVersion.md [hunk 1]
```diff
@@ -47,10 +47,19 @@ Used to store the server bucket encryption info
...
-```
-this._websiteConfiguration = websiteConfiguration || null;
+```javascript
+this._locationConstraint = LocationConstraint || null;
 ```
 
 ### Usage
 
-Used to store the bucket website configuration info
+Used to store the bucket website configuration info and the location constraint
```
Comment (LaurenSpiegel) [inline lib/metadata/ModelVersion.md]: this should just reference location constraint

File: lib/metadata/ModelVersion.md [hunk 2]
```diff
@@ -47,10 +47,19 @@ Used to store the server bucket encryption info
...
+of the bucket
+
+## Model version 5
+
+### Properties Added
+
+```javascript
+this._websiteConfiguration = websiteConfiguration || null;
+```
```
Comment (LaurenSpiegel) [inline lib/metadata/ModelVersion.md]: website config usage should be down here

File: lib/metadata/ModelVersion.md (hunk 1 above)
Comment (electrachong) [inline lib/metadata/ModelVersion.md]: @bennettbuchanan I went ahead and made this change
Thanks for spotting it @LaurenSpiegel

**Comment by electrachong** [comment]
Merging on behalf of @LaurenSpiegel

**Pull request merged**